alacritty/src/input.rs

```diff
@@ -469,31 +469,30 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
         match delta {
             MouseScrollDelta::LineDelta(_columns, lines) => {
                 let new_scroll_px = lines * self.ctx.size_info().cell_height;
-                self.scroll_terminal(new_scroll_px as i32);
+                self.scroll_terminal(new_scroll_px);
             },
             MouseScrollDelta::PixelDelta(lpos) => {
```

Comment:
Currently this returns an `lpos`, which means that it should be scaled by scale factor in theory.

I've sent a PR upstream to address this, so it should be fine for now:
https://github.com/rust-windowing/winit/pull/1483

Reply:
Once this is merged we shouldn't need any more `as f64` casts hopefully. :tada: